drivers/bluetooth/hci/rpmsg.c

```diff
@@ -0,0 +1,192 @@
+/*
```

Comment:
A file with this name already exists in the OpenAMP library, this could be a problem with some compilers (e.g. IAR)....  
Then i don't know if there are rules regarding the file and function namings...
I would suggest to rename it hci_rpmsg.c or rpmsg_hci.c  and function hci_rpmsg_*
But only a recommendation :)

Reply:
@arnopo , thanks for this note, actually :)
So, I would have no problem with renaming this to hci_rpmsg.c, this would solve your concern.
But then other HCI drivers here are named directly after the underlying transport, e.g. spic.c.
As long as we only care about GCC I see no problem.
@carlescufi your opinion?

Reply:
I see no problem at all with keeping the current name. I believe if we come to this problem we will need to solve it in a wider fashion across the  Zephyr codebase.

Reply:
Fair point. Actually @kapi-no had a suggestion to rename to rpmsg_hci.c. I have no strong opinion on this matter, as I said above.